src/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface RepairRequest {
```

Comment:
Zod can create an inferred type from a schema with `z.infer()`. Might be worth since you've opted for zod